ibis/bigquery/udf/tests/test_find.py

```diff
@@ -3,9 +3,15 @@
 import pytest
 
 from ibis.bigquery.udf.find import find_names
+from ibis.compat import PY38
 from ibis.util import is_iterable
 
-pytestmark = pytest.mark.bigquery
+if PY38:
```

Comment:
so as a followup, I'd like to change this to use the class based features rather than skips, e.g. e.g. in ibis/tests/backends, UDF can be modeled like for example RoundConvention (we explicitly add these tests to the backends that work with it).

please create an issue for this. thanks.

cc @icexelloss